CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Replace intermediate confirmation step in rollover protocol for the maker with a configurable flag which can be updated during runtime using the new `POST /rollover/config` endpoint.
   On startup, rollovers are configured to be accepted by default.
+- Breaking change: renamed `trading_pair` to `contract_symbol` in Http API.
```

Comment:
```suggestion
- Renamed `trading_pair` to `contract_symbol` in Http API.
```

🙃 At some point we should revisit our discussion about what a "breaking change" is. I would not state it like this in the changelog.
For us internally this is a breaking change, but for users it should not matter because the network protocols are not affected. 
If anybody looks at the changelog this looks like this version is incompatible to the one before, but that is misleading.

Reply:
Hmm, you are assuming that no one is trying to run their own `maker` automation then. And that this changelog is only meant to be consumed by users of the `taker` binary.

But I agree that our compatibility story is quite confusing.

Reply:
AFAICT; the PUT request didn't have `trading_pair` - it was only for the UI

Reply:
For me, the changelog is for developers but  yea, eventually we should revisit this discussion :)